kolibri/core/assets/src/views/AppBar.vue

```diff
@@ -134,36 +143,64 @@
       UiToolbar,
       KIconButton,
       SkipNavigationLink,
+      Navbar,
     },
-    mixins: [commonCoreStrings, navComponentsMixin],
+    mixins: [commonCoreStrings],
     setup() {
+      const store = getCurrentInstance().proxy.$store;
+      const $route = computed(() => store.state.route);
       const { windowIsLarge, windowIsSmall } = useKResponsiveWindow();
-      return { themeConfig, windowIsLarge, windowIsSmall };
+      const { topBarHeight, navComponents } = useNav();
+      const { isLearner, isUserLoggedIn, username, full_name } = useUser();
+      const links = computed(() => {
+        const currentComponent = navComponents.find(nc => nc.url === window.location.pathname);
```

Comment:
I don't see this used elsewhere in the file, so I am just wondering if it's used here, or if this syntax means it's now also available when `links` is referenced, i.e. in `<Navbar>`? I'm confused about how this works

Reply:
Lines 155 to 166 are defining the links computed prop, so `currentComponent` is only used inside the scope of the function that computes that.

Although, `currentComponent` is not really the right word now that they aren't components any more.